config/routes.rb

```diff
@@ -13,7 +13,7 @@
 
       resource :authorization, only: [:show] do
         resources :podcasts, except: [:new, :edit], module: :auth do
-          resources :episodes, except: [:new, :edit], module: :auth
```

Comment:
I confess I don't know enough about rails routes.rb to know why this change with no apparent controller change - is it just redundant because of the parent route?

Reply:
It was doubling the `auth` module ... so looking for the `Api::Auth::Auth::EpisodesController`.  (This route has never worked).

Reply:
Got it!

Reply:
Got it!